modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +44,63 @@ spec:
       istiod:
         type: PrivateKey
         privateKey:
-          rootCADir:  /etc/cacerts
+          rootCADir: /etc/cacerts
 ----
 
+[NOTE]
```

Comment:
Curious why you chose to make this a note rather than another step in the procedure?  I probably would have made it a step.  It's fine either way, I'm really just curious, did this feel like it needed more emphasis than a step?

Reply:
Yes, it was done for emphasis because in similar circumstances customer had issues in the field. They didn't always know when restating pods was essential.